src/mongocxx/test/spec/transactions.cpp

```diff
@@ -408,5 +412,7 @@ TEST_CASE("Transactions spec automated tests", "[transactions_spec]") {
         "TRANSACTIONS_TESTS_PATH", &run_transactions_tests_in_file, unsupported_transaction_tests);
 
     run_tests_in_suite("WITH_TRANSACTION_TESTS_PATH", &run_transactions_tests_in_file);
+
+    run_tests_in_suite("READ_WRITE_CONCERN_OPERATION_TESTS_PATH", &run_transactions_tests_in_file);
```

Comment:
I admit reusing the transactions test runner for this seems a little strange. But the format is the same. A unified test runner is in the near future (https://jira.mongodb.org/browse/DRIVERS-709). I think that will be the time to consolidate them all.

Reply:
I split this into a separate test runner.